hudi-common/src/main/java/org/apache/hudi/common/model/HoodieRecordMerger.java

```diff
@@ -48,6 +48,8 @@ public interface HoodieRecordMerger extends Serializable {
 
   String OVERWRITE_MERGER_STRATEGY_UUID = "ce9acb64-bde0-424c-9b91-f6ebba25356d";
 
+  String PAYLOAD_BASED_MERGER_STRATEGY_UUDID = "00000000-0000-0000-0000-000000000000";
```

Comment:
Let's add comprehensive docs on each UUID to explain what they are and how they are used in code.